### PR TITLE
Location Map + SFTP Scripts

### PR DESCRIPTION
**Pull request by chelsea-EYDS**
**What has changed:**

- added scripts to grab bcm and pcc files from sftp servers, parse, and drop in s3 
- slight reorganization of common dir
- fixed imports after implementing last step
- undid sales api update, created a new ticket, moved all previous PR comments to the new ticket
- added location mapping for parsed files

### CODE REVIEW

File: apps/backend/src/common/const/location-dictionary.ts [hunk 1]
```diff
@@ -0,0 +1,156 @@
+export const locationToMerchantMap = [
```
Comment (vyasworks) [inline apps/backend/src/common/const/location-dictionary.ts:1]: This is great, can we mark this under 'SBC' or add another key ?

File: apps/backend/src/common/const/location-dictionary.ts [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+  { office: 'Vanderhoof', merchant_terminal: 22044750, location_id: 92 },
+  { office: 'Vernon', merchant_terminal: 22044859, location_id: 93 },
+  { office: 'Victoria', merchant_terminal: 20777441, location_id: 94 },
+  {
+    office: 'Williams Lake',
+    merchant_terminal: 22044933,
+    location_id: 97
+  },
+  {
+    office: 'SBC\u00a0Mobile\u00a0GVA',
```
Comment (vyasworks) [inline apps/backend/src/common/const/location-dictionary.ts:122]: Is this unicode character `\u00a0` intentional?

File: apps/backend/src/flat-files/tdi17/TDI17Details.ts [hunk 3]
```diff
@@ -17,10 +17,10 @@ export interface ITDI17Details extends IFixedWidthRecord<ITDI17Details> {
   deposit_time: string;
   seq_no: string;
   location_desc: string;
-  deposit_amt_curr: number;
+  deposit_amt_curr: string;
```
Comment (vyasworks) [inline apps/backend/src/flat-files/tdi17/TDI17Details.ts:20]: why is this changing to string?

File: apps/backend/src/temp-scripts/sftp.garms.sh [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# ! /bin/bash
```
Comment (vyasworks) [inline apps/backend/src/temp-scripts/sftp.garms.sh:1]: add readme on deets, how to use, export vars, where to find them in our sftp. whom to reach out for pvt keys.

**Review by vyasworks — APPROVED** [review]

File: apps/backend/src/flat-files/tdi17/TDI17Details.ts (hunk 3 above)
Comment (chelsea-EYDS) [reply apps/backend/src/flat-files/tdi17/TDI17Details.ts:20]: For ease of testing. This way we can ensure the correct number of characters in the string. I had planned to parse it while working with the data later, and to just keep this value exactly as is until we are ready to work with the data.

**Review by vyasworks — APPROVED** [review]

**Pull request merged**